Gemfile.lock

```diff
@@ -51,8 +58,8 @@ GEM
     html-pipeline (2.3.0)
       activesupport (>= 2, < 5)
       nokogiri (>= 1.4)
-    i18n (0.7.0)
```

Comment:
when trying to build locally, getting the error `Your Gemfile.lock is corrupt. The following gem is missing from the DEPENDENCIES section: 'i18n'` - guessing it may have something to do with this. I'm going to try blowing away and recreating the lock file so that everything re-resolves.